cpp/benchmarks/iterator/iterator_benchmark.cu

```diff
@@ -198,7 +202,7 @@ void BM_pair_iterator(benchmark::State &state)
   cudf::column_view hasnull_F = wrap_hasnull_F;
   cudf::column_view hasnull_T = wrap_hasnull_T;
 
-  rmm::device_vector<thrust::pair<T, bool>> dev_result(1, {T{0}, false});
+  auto dev_result = cudf::detail::make_zeroed_device_uvector_sync<thrust::pair<T, bool>>(1);
```

Comment:
I feel like this should have a comment that says this initializes `dev_result` to _all false_.

Reply:
Added.